liquid_node/commands.py

```diff
@@ -450,6 +474,110 @@ def deletecollection(name):
 
 
+def importcollection(name, path, method='copy'):
+    """Import a single collection extracted collection.
+    The path must contain:
+    1. The blobs folder, named `blobs`
+    2. The database folder, named `pg`, containing a subfolder called `data`.
+    3. the index, named <name>-index.tgz
+
+    :param name: name of the collection you want to import
+    :param path: path to the collection data.
+    :param method: can be 'copy', 'move' or  'link'. Copy is default.
+    """
+    path = Path(path).resolve()
+    database = path / 'pg'
+    blobs = path / 'blobs'
+    index = path / f'{name}-index.tgz'
+    log.info(f'Importing collection {name}')
+    # parsing `method`, which is not interpreted as a kwarg
+    if method != 'copy':
+        _, method = method.split('=', 1)
+        if method not in ['copy', 'move', 'link']:
+            raise RuntimeError(f'unknown method: {method}')
+
+    node_name = name.lower()
+    if name != node_name:
+        log.info(f'Renaming "{name}" to "{node_name}"')
+    collection_path = Path(config.liquid_volumes) / 'collections' / node_name
+    if collection_path.exists():
+        if confirm(f'collection {name} already exists, overwrite (hoover needs to be running)?'):
+            redeploy = True
+            purge_collection(name)
+        else:
+            log.info(f'exiting and not overwriting data for {name}')
+            exit()
+    else:
+        redeploy = False
+        (collection_path).mkdir(parents=True)
+
+    # copy the pg dir
+    database = Path(database).resolve(strict=True)
+    if not (database / 'data' / 'PG_VERSION').exists():
+        raise RuntimeError("database is not a valid Postgres database")
+    pg_src = database
+    pg_dst = collection_path / 'pg'
+    import_dir(pg_src, pg_dst, method=method)
+
+    # copy the blobs dir
+    blobs = Path(blobs).resolve(strict=True)
+    blob_src = blobs
+    blob_dst = collection_path / 'blobs'
+    import_dir(blob_src, blob_dst, method=method)
+    (collection_path / 'index').mkdir()
+    # copy or link the index
+    index = Path(index).resolve(strict=True)
+
+    index_dst = collection_path / 'index' / f'{name}-index.tgz'
+    import_file(index, index_dst, method=method)
+
+    log.info(f'Imported collection using method: {method}')
+    if redeploy:
+        print(f'redeploy in order to make changes to {name} available.')
+    else:
+        print('add the following line to liquid.ini')
+        print(f'[collection:{name}]')
+
+
+def exportcollection(name):
+    """Export a collection from node. Liquid has to be running.
+
+    :param name: name of the collection you want to export.
+
+    """
+    if name not in config.collections:
+        raise RuntimeError(f'Collection {name} does not exist in the liquid.ini file.')
+    export_path = Path(config.liquid_collections) / 'exported' / name
+    if export_path.exists():
+        if confirm(f'collection {name} already has exported files in {export_path}, overwrite?'):
```

Comment:
How about we just print an error and bail? It's easy to `sudo rm -rf the/export/path` and try again, and we reduce some complexity.

Reply:
I thought it would be nicer and maybe we want to confirm other stuff at some point as well. I had to import/export a lot of test collections over and over again so that felt quicker. But I don't mind deleting it.